pkg/test/resourcefixture/testdata/basic/apphub/v1alpha1/apphubapplication/apphubapplication-minimal/update.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: apphub.cnrm.cloud.google.com/v1alpha1
+kind: AppHubApplication
+metadata:
+  name: apphubapplication-minimal-${uniqueId}
+spec:
+  parent:
```

Comment:
I like this but the existing convention is that we don't show `spec.parent`. It would be `spec.projectRef`